app/mailers/invitation_mailer.rb

```diff
@@ -8,17 +8,23 @@ class InvitationMailer < ApplicationMailer
 
   default from: ENV['FROM_EMAIL_ADDRESS']
 
-  def invitation_mail(email_address, message, invitation_url)
+  def invitation_mail(email_address, message, invitation_url, template)
     @invitation_url = invitation_url
     @message = message
-    mail(subject: DEFAULT_INVITATION_SUBJECT, to: email_address)
+    # Use the default template if the given template does not exist.
+    template = lookup_context.find_all(template, []).blank? ? 'invitation_mailer/invitation_mail' : template
+    mail(subject: DEFAULT_INVITATION_SUBJECT, to: email_address) do |format|
+      format.html { render template: template }
+    end
   end
 
+  # Only used for u-can-act symposium registration. Remove when removing the u-can-act project.
```

Comment:
maybe prepend a TODO so it's easier to find in the future?

Reply:
Fixed.